test/lazy/test_reuse_ir.py

```diff
@@ -37,7 +37,8 @@ def testAdd(self):
             torch._lazy.mark_step()
 
         torch.testing.assert_close(z.cpu(), z_lazy.cpu())
-        assert metrics.counter_value("IrNodeReused_torch::lazy::AddTensor") >= 16
+        print(metrics.counter_value("IrNodeReused_torch::lazy::AddTensor"))
+        assert metrics.counter_value("IrNodeReused_torch::lazy::AddTensor") >= 16 - 2
```

Comment:
Do you mean to keep `16-2` this way as a reminder for reusing SizeNode later? If that's the case, then better to add comments here.